components/nav-avatar.tsx

```diff
@@ -124,7 +126,7 @@ export default function NavAvatar() {
         </div>
 
         <LinksSession>
-          {internalLinks.map(ProfileInternalLink)}
+          {handleLinksWithoutNetwork(internalLinks, !!query?.network).map(ProfileInternalLink)}
```

Comment:
This links list is being repeated on both components, so I think that a better solution could be to refactor this list into something like this [navigation-links.ts](https://github.com/layerx-labs/webnetwork/blob/dev/helpers/navigation-links.ts) and use it for both `nav-avatar.tsx` and `profile-links.tsx` components. This way we can make it easier to change and maintain.